packages/components/src/Radio/Radio/_docs/Radio.stories.tsx

```diff
@@ -6,8 +6,20 @@ const meta = {
   component: Radio,
   args: {
     id: "radio",
-    name: "Radio",
-    value: "Radio",
+    name: "name",
+    value: "value",
+  },
+  argTypes: {
+    name: {
```

Comment:
I was able to use JSdocs by declaring the prop to be the same as the HTMLAttribute type signature, then putting the comment above. Storybook seemed to infer fine. Worth a try again, if that's what you aimed to achieve (sometimes SB caches, so that also might be why it didn't seem to work).

Reply:
Thanks, I'll have another go! If it still doesn't work, can I hit you up for a quick pair?

Reply:
Yep no problem!